azure_media_services/static/js/player.js

```diff
@@ -179,21 +163,42 @@ function initTranscript(player, transcript, transcriptPaneEl) {
     console.log(error);
   }
 
-  parser.parse(transcript);
+  try {
+    parser.parse(transcript);
+  }
+  catch (e) {
+    //todo:remove when firefox bug is fixed.
```

Comment:
not a true fix and we don't plan on shipping this part publicly. this messaging will reduce feedback on a known issue when we advertise this xblock for internal testing